qulice-checkstyle/pom.xml

```diff
@@ -103,6 +103,12 @@
                     <path>${project.artifactId}</path>
                 </configuration>
             </plugin>
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <argLine>-Duser.language=en</argLine>
```

Comment:
@mkordas I would also add `-Duser.country=US` (to minimize potential problems with thousands separator, or decimals)